root/static/scripts/common/components/EntityLink.js

```diff
@@ -215,8 +216,17 @@ $ReadOnlyArray<Expand2ReactOutput> | Expand2ReactOutput | null => {
     anchorProps.title = hover;
   }
   content = disableLink
-    ? <span className="deleted">{isolateText(content)}</span>
-    : <a key="link" {...anchorProps}>{isolateText(content)}</a>;
+    ? (
+      <span
+        className="deleted"
+        title={isGreyedOut(content)
+          ? l(`This link has been temporarily disabled because
+            it has been reported as potentially harmful.`)
+          : null}
+      >
+        {isolateText(content)}
+      </span>
```

Comment:
In this specific case, the link might not be harmful but just linked via an “ended” relationship.

Reply:
My bad, didn’t see the `isGreyedOut` test.